src/docs/cursor-ops.dox

```diff
@@ -161,4 +161,45 @@ that may not be modified or freed by the application.  If a longer scope
 is required, the application must make a copy of the memory before the
 cursor is re-used, closed or reset.
 
+@section cursor_bound Apply bounds to the cursor
+
+Cursors may have bounds applied to them. This sets a start and stop key for
+the cursor to operate within, known as the lower and upper bounds.
+
+```
+cursor->set_key(cursor, A)
+cursor->bound(cursor, bound=lower)
+```
+
+Both or either one of the lower and upper bounds can be set, and the unset bound will be assumed to be off the start or
+end of the key range accordingly. When setting the bounds, the relevant bound is set prior with cursor->set_key.
+This key does not have to exist or be visible in order for bounds setting to succeed.
+
+Cursor operations next, prev, search, and search near support bounds. When performing a traversal such as cursor->next
```

Comment:
We actually also check bounds in cursor `modify` `insert` `update` `remove`

Reply:
[6b88eb1](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/6b88eb16841dd50de96ea1436b961ec3a0e8644d)